tools/pd-simulator/simulator/cases/import_data.go

```diff
@@ -140,6 +143,37 @@ func newImportData() *Case {
 		if checkCount > uint64(getRegionNum())/10 {
 			isEnd = dev < 0.002
 		}
+		if isEnd {
+			var rangeColor = []string{
+				"#313695", "#4575b4", "#74add1", "#abd9e9", "#e0f3f8",
+				"#fee090", "#fdae61", "#f46d43", "#d73027", "#a50026",
+			}
+			bar3d := charts.NewBar3D()
+			bar3d.SetGlobalOptions(
+				charts.TitleOpts{Title: "New region count"},
+				charts.VisualMapOpts{
+					Range:      []float32{0, float32(getRegionNum()) / 10},
+					Calculable: true,
+					InRange:    charts.VMInRange{Color: rangeColor},
+					Max:        float32(getRegionNum()) / 10,
+				},
+				charts.Grid3DOpts{BoxDepth: 80, BoxWidth: 200},
+			)
+			xAxis := make([]int, 10)
+			for i := 1; i <= 10; i++ {
+				xAxis[i-1] = i
+			}
+			yAxis := make([]int, checkCount)
+			for i := 1; i <= int(checkCount); i++ {
+				yAxis[i-1] = i
+			}
+			bar3d.AddXYAxis(xAxis, yAxis).AddZAxis("bar3d", newRegionCount)
+			f, _ := os.Create("region_3d.html")
```

Comment:
can we show the total region in another file?

Reply:
Now supported.
![image](https://user-images.githubusercontent.com/41138939/93182506-59012f80-f76c-11ea-8c20-7b9a43a0e424.png)

Reply:
Can we change the label of each axis?

Reply:
Their API now can only add data, no `set title` interface...